### DISCUSSION TITLE
Particle Effects based on perms

### DISCUSSION THREAD

**Discussion by 1ntent1on** [Zrips/CMI#7505] [Feature Requests]
Hi, it would be really nice to able to set required perms for each TeleportEffect in the particle section and to be able to completely enabled/disable them via perms for some players.

Good example usage of this would be that with proper LP context TeleportEffects could be disabled for Vanished players etc. Also it would be a nice EULA friendly perk for players.

**Comment by CloudeLecaw** [comment]
nice idea but its missleading the fact this should be a thing for all particles because your defined effects are particles too.
lets see what the #notes about TeleportEffects say..
```
# Defined animated particles will be shown on player teleportation on particular action from place and to destination
# if you don't want it to be shown then set it to empty field
# Animation names should be used from Particles section from above like TpUp
``` 

well looks like this was a little hint that these also are just particles...
so if this would be added make it for ALL particles of the config ...not just the Teleportparticles
